backend/src/main/java/com/redhat/cloud/notifications/models/InternalRoleAccess.java

```diff
@@ -93,7 +93,9 @@ public boolean equals(Object o) {
         }
         if (o instanceof InternalRoleAccess) {
             InternalRoleAccess other = (InternalRoleAccess) o;
-            return Objects.equals(id, other.id);
+            return Objects.equals(id, other.id)
```

Comment:
Usually comparing only the id should be enough, it shouldn't be possible for 2 objects with the same id to be different

Reply:
All right, reverted back! Thanks!